erpnext/crm/report/opportunity_summary_by_sales_stage/opportunity_summary_by_sales_stage.js

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2016, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+/* eslint-disable */
+
+frappe.query_reports["Opportunity Summary by Sales Stage"] = {
+	"filters": [
+		{
+			fieldname:"based_on",
+			label: __("Based On"),
+			fieldtype:"Select",
+			options:"Opportunity Owner\nSource\nOpportunity Type",
+			default:"Source"
+		}
+		,
+		{
+			fieldname:"data_based_on",
+			label: __("Data Based On"),
+			fieldtype:"Select",
+			options:"Number\nAmount",
+			default:"Number"
+		}
+		,
+		{
+			fieldname:"from_date",
+			label: __("From Date"),
+			fieldtype: "Date",
+
+		}
+		,
+		{
+			fieldname:"to_date",
+			label: __("To Date"),
+			fieldtype: "Date",	
+		}
+		,
+		{
+			fieldname:"status",
+			label: __("Status"),
+			fieldtype:"MultiSelectList",
+			get_data:function(){
```

Comment:
```suggestion
			get_data: function() {
```
Check for this formatting issue everywhere.